draft-irtf-cfrg-hpke.md

```diff
@@ -710,11 +711,54 @@ byte strings for public keys.
 | 0x0002 | AES-GCM-256      | 32  | 12  | {{GCM}}      |
 | 0x0003 | ChaCha20Poly1305 | 32  | 12  | {{?RFC8439}} |
 
-# Security Considerations
-
-The general security properties of HPKE are described in
-{{security-properties}}.  In this section, we consider a security issue that may
-arise in practice and an advanced use case.
+# Security Considerations {#sec-considerations}
+
+HPKE makes the following assumptions about its cryptographic
+dependencies.
+
+- Hash: Collision resistance.
+- Extract: Indifferentiable from a Random Oracle.
+- Expand: Behaves as a Pseudorandom Function wherein the first
```

Comment:
Pseudo-Random Function

Reply:
I might actually change this to "pseudorandom function", as that's what was done in RFC8446!

Reply:
ack! I was just suggesting this because you had Random Oracle capitalized. If you do “pseudorandom function”, then maybe also “random oracle”